spec/plugin_spec_helper.rb

```diff
@@ -21,19 +21,19 @@ module OpenProject
   module GlobalRoles
     module PluginSpecHelper
       def mobile_tan_plugin_loaded?
-        plugin_loaded?("openproject_mobile_otp")
+        plugin_loaded?('openproject_mobile_otp')
       end
 
       def privacy_plugin_loaded?
-        plugin_loaded?("openproject_dtag_customizing")
+        plugin_loaded?('openproject_dtag_customizing')
       end
 
       def costs_plugin_loaded?
-        plugin_loaded?("openproject_costs")
+        plugin_loaded?('openproject_costs')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.